app/src/main/java/com/example/keybindhelper/ui/keybind/KeybindFragment.kt

```diff
@@ -19,79 +19,80 @@ import com.example.keybindhelper.dao.CurrentProjectManager
                 println("LOADED")
-                loadRecycleView(root);
+                loadRecycleView(root)
             }
-        }else{
+        } else {
             println("already loaded")
-            loadRecycleView(root);
+            loadRecycleView(root)
         }
 
 
         return root
     }
 
-    private fun initMenu(mainActivity: MainActivity,view: View) {
-        mainActivity.setAppBarTitle(CurrentProjectManager.CurrentProject!!.name!!);
+    private fun initMenu(mainActivity: MainActivity, view: View) {
+        mainActivity.setAppBarTitle()
         mainActivity.showMenuItems(mainActivity.keybindsFragmentActionMenuIds)
-        val rv=view.findViewById<RecyclerView>(R.id.recyclerView);
+        val rv = view.findViewById<RecyclerView>(R.id.recyclerView)
 
-        mainActivity.Menu!!.findItem(R.id.action_delete_all_groups).setOnMenuItemClickListener {
-            val cd=ConfirmDialog(view.context,"Delete All Groups?")
-            cd.onConfirmed = object : ConfirmDialog.ConfirmedEvent{
+        mainActivity.Menu.findItem(R.id.action_delete_all_groups).setOnMenuItemClickListener {
+            val cd = ConfirmDialog(view.context, "Delete All Groups?")
+            cd.onConfirmed = object : ConfirmDialog.ConfirmedEvent {
                 override fun onConfirmed() {
                     CurrentProjectManager.deleteAllGroups()
                     rv.adapter = GroupAdapter(CurrentProjectManager.Groups)
                 }
             }
             cd.Show()
 
-            true;
+            true
         }
-        mainActivity.Menu!!.findItem(R.id.action_sub_hide_keybinds).setOnMenuItemClickListener {
-            for(v in rv.children)
-                v.findViewById<RecyclerView>(R.id.keybind_zone).isVisible = false;
-            true;
+        mainActivity.Menu.findItem(R.id.action_sub_hide_keybinds).setOnMenuItemClickListener {
+            for (v in rv.children)
+                v.findViewById<RecyclerView>(R.id.keybind_zone).isVisible = false
+            true
         }
-        mainActivity.Menu!!.findItem(R.id.action_sub_show_keybinds).setOnMenuItemClickListener {
-            for(v in rv.children)
-                v.findViewById<RecyclerView>(R.id.keybind_zone).isVisible=true;
-            true;
+        mainActivity.Menu.findItem(R.id.action_sub_show_keybinds).setOnMenuItemClickListener {
+            for (v in rv.children)
+                v.findViewById<RecyclerView>(R.id.keybind_zone).isVisible = true
+            true
         }
-        mainActivity.Menu!!.findItem(R.id.action_add).setOnMenuItemClickListener {
+        mainActivity.Menu.findItem(R.id.action_add).setOnMenuItemClickListener {
             CurrentProjectManager.AddGroup()
-            System.out.println("MainActivity.floatingactionbutton.click: Groups Size:" + CurrentProjectManager.Groups!!.size)
+            println("MainActivity.floatingactionbutton.click: Groups Size:" + CurrentProjectManager.Groups!!.size)
```

Comment:
println is enough for Kotlin
source: https://kotlinlang.org/docs/basic-syntax.html#print-to-the-standard-output

Reply:
Use proper logging instead of println().
- Logging has the concept of logging levels.  They can be filtered and aggregated based on those levels.
- No one will see println().